src/map/entities/trustentity.h

```diff
@@ -0,0 +1,45 @@
+﻿/*
+===========================================================================
+
+  Copyright (c) 2018 Darkstar Dev Teams
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see http://www.gnu.org/licenses/
+
+  This file is part of DarkStar-server source code.
+
+===========================================================================
+*/
+
+#ifndef _CTRUSTENTITY_H
+#define _CTRUSTENTITY_H
+
+#include "mobentity.h"
+
+class CCharEntity;
+class CTrustEntity : public CMobEntity
+{
+public:
+    CTrustEntity(CCharEntity*);
+	~CTrustEntity();
+	uint8 m_Element;
+	uint32 m_PetID;
```

Comment:
im not sure we want these public, instead we should make these privte and use set and get
e.g.
```cpp
uint32 getPetID() const;
uint8 getElement() const;
```

Reply:
i would personally prefer modifying members through get/set instead of leaving them exposed for the sake of readability and debugging purposes, even if it results in additional code
i guess go with whatever @teschnei wants

Reply:
Waiting feedback then.

Reply:
it's not a big enough deal for me either way, let's just leave it the way it is